daisy_workflows/export/export_disk_ext.sh

```diff
@@ -19,17 +19,10 @@ GS_PATH=$(curl -f -H Metadata-Flavor:Google ${URL}/gcs-path)
 FORMAT=$(curl -f -H Metadata-Flavor:Google ${URL}/format)
 
 # Strip gs://
-GCS_PATH=${GS_PATH##*//}
-# Grab only the bucket
-GCS_BUCKET=${GCS_PATH%%/*}
-# Mount GCS bucket containing the disk image.
-mkdir -p "/gcs/${GCS_BUCKET}"
-gcsfuse ${GCS_BUCKET} /gcs/${GCS_BUCKET}
-if [ $? -ne 0 ]; then
-  echo "ExportFailed: Failed to mount ${GCS_BUCKET}."
-fi
-OUTS=${GCS_PATH%/*}
-mkdir -p "/gcs/${OUTS}"
+LOCAL_PATH=${GS_PATH##*//}
+# Create dir for output
+OUTS=${LOCAL_PATH%/*}
```

Comment:
what is $OUTS used for?

Reply:
to create dir for outputs in below line. let me rename it to "OUTS_PATH" to clarify.